content/building-blocks/configure-your-computer/infrastructure-choice/google-colab.md

```diff
@@ -1,29 +1,67 @@
----
-title: "Use Google Colab to Run Python in the Cloud"
-description: "Got a slow computer? Or want to collaborate on Python code? Use Colab!"
-keywords: "cloud, python, jupyter, slow computer, fast, collaborate"
-date: 2022-04-08T10:01:14+05:30
-draft: false
-author: "Cas Ruger"
-# authorlink: "https:/"
-aliases:
-  - /use/colab
-  
----
-
-## Overview
-
-
-## How to get started
-
-{{% warning %}}
-
-Some warning
-
-{{% /warning %}}
-
-
-## Advanced use cases
-
-### Adding collaborators
-
+---
+title: "Use Google Colab to Run Python in the Cloud"
+description: "Got a slow computer? Or want to collaborate on Python code? Use Colab!"
+keywords: "cloud, python, jupyter, slow computer, fast, collaborate, google"
+date: 2022-04-08T10:01:14+05:30
+draft: false
+author: "Cas Ruger"
+# authorlink: "https:/"
+aliases:
+  - /use/colab
+  
+---
+
+## What is Google Colab
+Colab is a product from Google Research that allows anybody with an internet connection to write and execute python code through a browser. Colab is free to use, and there is no setup on your computer needed. It is especially useful if you have a slow computer since Google hosts your Jupyter notebook and thus uses their GPU free of charge. Or if you want to work together on the same code, since you can collaborate on the same cloud file with multiple editors. 
+
+## How to get started
+It is time to create your own Colab notebook.
+
+
+1. Go to https://colab.research.google.com/ 
+2. Press **New notebook**, or if you already have a file, choose this one from the list
+3. Save your notebook to either Google drive or GitHub via **File** > **Save a copy in ...**
+
+
+## How to use Colab
+Once you are in your Colab notebook you can use it just like any offline Jupiter notebook. 
+
+Type your code in the darker gray box en pres the run arrow to run the code. Your output will be placed below the box. To create a new code box, press **+ Code** in the navigation bar. 
+
+
+### Import packages
+Colab environment already has a number of pre-installed packages such as pandas, pytorch, scipy, tenserflow and numpy. To check all installed packages use **pip freeze** 

```

Comment:
Wrap package names in `` so it is rendered as code that can be copy pasted, e.g.,

`pandas`, `pytorch`, `scipy`, etc.